grammars/javascript.cson

```diff
@@ -724,6 +724,41 @@
     ]
   }
   {
+    # Math
+    'begin': '\\bMath\\b'
```

Comment:
`\b` matches word boundaries; it doesn't match $ (tested in Atom 1.7.0). Both JavaScript and Oniguruma regular expressions classify `\w` characters as `[A-Za-z0-9_]`.

`\\bMath\\b` is consistent with `\\bconsole\\b` which is already in the code.